data/cncf.yaml

```diff
@@ -4624,14 +4681,17 @@
       check_sets:
         - code
         - community
+        - docs
```

Comment:
The `docs` check set is a subset of the `code` check set, so it isn't needed in this case. Please remove it.